src/API/helpers.js

```diff
@@ -1,2 +1,3 @@
 export const provisioningUrl = (path) => `/api/provisioning/v1/${path}`;
 export const imageBuilderURL = (path) => `/api/image-builder/v1/${path}`;
+export const LIMIT = 100;
```

Comment:
@akhil-jha here we've set the final batch, tho agreed the example is missleading, but it would be hard to create 100+ keys to showcase it.

Reply:
Ahh. Okay. :)